csharp-refactoring/RefactoringCouplers/RefactoringCouplers/FeatureEnvy/Correct/Customer.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace RefactoringCouplers.FeatureEnvy.Correct;
+public class Customer
+{
+    public Customer(string name)
+    {
+        this.Name = name;
```

Comment:
No need to use the `this` keyword since there is a difference between the property and the parameter (the first letter is capitalized for the property). You also don't use it in your other classes, so it would be better if we are consistent 😄